### ISSUE TITLE
[Accessibility][Toolbar] The toolbar does not follow the established design pattern

### ISSUE THREAD

**Issue by jmbuss** [awslabs/iot-app-kit#2510]
**Describe the bug**

**Standard:** 4.1.2 Name, Role, Value (A)
**Affected Groups:** Screen Reader Users, Keyboard Users
 
**Notes and Advice** 
The dashboard toolbar does not follow the established design pattern.
 
The dashboard toolbar has the following issues: No accessible name (fails [SC 4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG/#name-role-value))

**Recommendation** 
Provide an accessible name for the toolbar container using aria-labelledby (if there already is a visible heading or label), or aria-label (if there is no visible label).
If there is more than one toolbar on the page, use aria-label to give each a unique accessible name.

1. We should have a container which labels the widgets toolbar
2. We should have a container which labels the dashboard actions (save / preview)
3. We should have a section which labels the dashboard query configuration (asset model select view mode only / viewport picker)

**Acceptance Criteria**
The above violation is mitigated
